src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/utils/validation.ts

```diff
@@ -113,6 +124,22 @@ export const errorMessageKeys = {
     textKey: 'multipleOf',
     paramKey: 'multipleOf',
   },
+  oneOf: {
+    textKey: 'oneOf',
+    paramKey: 'passingSchemas',
+  },
+  anyOf: {
+    textKey: 'anyOf',
+    paramKey: 'passingSchemas',
+  },
+  allOf: {
+    textKey: 'allOf',
+    paramKey: 'passingSchemas',
+  },
+  not: {
```

Comment:
not sure what the 'paramKey' does, but are we actually going to support 'not' at this point?

Reply:
`not` is not currently supported in our generator tool, but the app-frontend json schema validation should be agnostic and just validate as if it is supported. When we start supporting this in our studio tool we should agree on some standard mesages and such if this validation fails.